tests/components/universal/test_media_player.py

```diff
@@ -681,6 +681,184 @@ def test_supported_features_children_and_cmds(self):
 
         assert check_flags == ump.supported_features
 
+    async def test_overrides(self):
+        """Test overrides."""
+        config = copy(self.config_children_and_attr)
+        await async_setup_component(self.hass, "media_player", config)
+        excmd = {"service": "media_player.test", "data": {"entity_id": "test"}}
```

Comment:
`"test"` is not a valid entity_id. I'd use a valid format.
```suggestion
        excmd = {"service": "media_player.test", "data": {"entity_id": "media_player.overridden"}}
```